ebos/eclmpiserializer.hh

```diff
@@ -21,7 +21,6 @@
 #ifndef ECL_MPI_SERIALIZER_HH
 #define ECL_MPI_SERIALIZER_HH
 
-#include <dune/common/version.hh>
 #include <opm/simulators/utils/ParallelRestart.hpp>
```

Comment:
Does this implicitly rely on `ParallelRestart.hpp` also bringing in `ParallelCommunication.hpp`?  If so, I think the requirement  should be made explicit by including `ParallelCommunication.hpp` directly.

Reply:
done